pl_bolts/datamodules/vision_datamodule.py

```diff
@@ -30,6 +27,9 @@ def __init__(
         shuffle: bool = True,
         pin_memory: bool = True,
         drop_last: bool = False,
+        train_transforms: Optional[Callable] = None,
```

Comment:
Shouldn't we include these transforms into the class docstring?

Reply:
I agree we should add it. What will be an ideal way of doing this? Do I need to create another pull request for this?

Reply:
I agree with this as well, good catch @luca-medeiros!

@shivammehta007, yes, opening a new PR is the only option (not even I have a push rights to master :joy:). Since this PR is already merged, there really is no other option. Just write there it's a followup of #843

Reply:
@otaj 

Seems you are the only member doing reviews for #839. 
What about assembling a small (3~5 people) team to help you out with it? I would be willing to help review as well!

Reply:
@luca-medeiros Oh, absolutely! That's the whole point of our Slack channel. I think you're not in there yet, can you ping me on PL Slack (`@Ota`) and I will add you there?